### PR TITLE
[hawkular-1259] Fixing all tests

### PR DESCRIPTION
**Pull request by jotak**
- Update cassettes with new docker image
- The new docker image fixes inventory/status endpoint issue and brings back authentication by default
- Now using a fixed admin token for tests

### CODE REVIEW

**Comment by jotak** [comment]
Note: this PR is based on #240 , which should be merged first, and then this one rebased
Commits displayed here include commits of #240.

**Comment by jotak** [comment]
PS: travis is lying. It works :)

File: spec/integration/metric_spec.rb [hunk 1]
```diff
@@ -661,7 +661,7 @@
             end
 
             data = @client.gauges.get_data(hawkular_mem_id)
-            expect(data.size).to be > 2 # needs the services to be running for more than 2 minutes
+            expect(data.size).to be > 2 # needs the services to be running for some time (~15 min)
```
Comment (josejulio) [inline spec/integration/metric_spec.rb]: I'm not sure I understand this comment:
`# needs the services to be running for some time (~15 min)`
Yet the live tests are passing in under 10 minutes.

**Comment by josejulio** [comment]
> PS: travis is lying. It works :)

A good kick made it reconsider.

File: spec/integration/metric_spec.rb (hunk 1 above)
Comment (jotak) [reply spec/integration/metric_spec.rb]: Interesting... I think collection interval for OS metrics is set to 5 minutes... maybe there's some specific config for travis?

Comment (jotak) [reply spec/integration/metric_spec.rb]: Just confirming, it's the purpose of script file in `.travis/build_config_for_testing.rb` So, when cassettes are generated locally without any specific configuration, it's about 15 minutes to wait for >2 datapoints of this metric.

**Comment by jshaughn** [comment]
This looks good to me.  Not sure why it is not yet merged as we need to get the 1259 work completed.  I'll merge now, follow-on work can be done in other PR's.

**Pull request merged**